backend/inspirehep/utils.py

```diff
@@ -32,7 +32,7 @@ def chunker(iterable, chunksize):
 
 
 def flatten_list(input_list):
-    if isinstance(input_list, list):
+    if isinstance(input_list, list) or isinstance(input_list, tuple):
```

Comment:
```suggestion
    if isinstance(input_list, (list, tuple)):
```

Reply:
This I should actually revert as this change is not needed anymore.

Reply:
Ah I'm still using it in one test. I will update it as you suggest :) :+1: